pkg/resource/deploy/step_generator.go

```diff
@@ -304,8 +299,9 @@ func (sg *stepGenerator) GenerateSteps(event RegisterResourceEvent) ([]Step, res
 				urn = s.URN()
 			}
 			if resourcePlan, ok := sg.deployment.newPlans.get(urn); ok {
-				// If the resource is in the plan, add the operation to the plan.
```

Comment:
This comment is still correct?

Reply:
Will add back. Was advised that obvious/redundant comments are bad and I added it last time so, in passing, I thought I'd clean it up.